addons/pos_self_order/controllers/entry.py

```diff
@@ -30,39 +26,38 @@ class PosQRMenuController(http.Controller):
             ],
         auth="public", website=True, sitemap=True,
     )
-    def pos_self_order_start(self, config_id=None, access_token=None, table_identifier=None):
```

Comment:
Let's move the controllers that involve `/menu` to pos_restaurant_self_order. Later, for the kiosk, we'll just introduce a new endpoint, perhaps called `/kiosk`, to render the kiosk app.